clusters/app.ci/ci-chat-bot/ci-chat-bot.yaml

```diff
@@ -185,6 +223,11 @@ spec:
       containers:
       - name: bot
         image: ci-chat-bot:latest
+        ports:
+          - name: main
+            containerPort: 8080
+          - name: metrics
+            containerPort: 9090
```

Comment:
These are mostly informative fields that can safely be omitted.

Reply:
Removed.